bokeh/models/widgets/inputs.py

```diff
@@ -160,6 +160,15 @@ class FileInput(Widget):
         A valid `IANA Media Type`_, with no parameters.
 
     .. _IANA Media Type: https://www.iana.org/assignments/media-types/media-types.xhtml
+
+    .. note::
+        A bug in some versions of Chrome on macOS Big Sur currently limits
```

Comment:
if you wanted to hedge this against forgetting to remove it, suggest "may limit" instead of "currently limits"

Reply:
Another great idea! 😀